docs/linter.md

```diff
@@ -12,9 +12,9 @@ The linter should make it easier to enforce team coding standards by providing g
 
 - enable/disable analyzer
 - set rule-specific values (e.g. DisallowedHosts for [`no-hardcoded-env-urls`](./linter-rules/no-hardcoded-env-urls.md) rule)
-- set DiagnosticLevel of rules:
+- set level of rules:
 
-| **DiagnosticLevel**  | **Build-time behavior** | **Editor behavior** |
+| **level**  | **Build-time behavior** | **Editor behavior** |
```

Comment:
Not sure this chart is necessary.

Reply:
The Roslyn analyzer main docs site uses the same chart. I find it helpful.

Reply:
https://docs.microsoft.com/en-us/visualstudio/code-quality/roslyn-analyzers-overview?view=vs-2019#severity-levels-of-analyzers